pages/public_cloud/public_cloud_network_services/configuration-04-update_subnet/guide.en-gb.md

```diff
@@ -0,0 +1,159 @@
+---
+title: Update a subnet options
+excerpt: Find out how to update the options of an existing subnet
+updated: 2024-01-19
+---
+
+## Objective
+
+This page explains how to update the options of an existing subnet. This can be needed if you want to comply with the pre requisites to be able to use the Public Cloud Gateway : have a gateway IP defined in your subnet.
+For this example, we will use a private network and a subnet with the following attributes :
+- CIDR 10.1.0.0/16
+- DHCP enabled with allocation pool on the global range
+- No gateway IP 
+
+We will see how to update it to have a gateway IP set to 10.1.0.1 and remove that IP from the allocation pool. 
+
+
+## Prerequisites
+You shall have created a private network and a subnet as explained in that [guide](../getting-started-07-creating-vrack/guide.en-gb.md)
+
+
+## Instructions
+
+### OVHCloud Control Panel
+It is currently not possible to update a subnet through this interface
+
+### Horizon
+- Connect to Horizon & choose the region where the subnet is defined as explained in that [guide](../../compute/introducing_horizon/guide.en-gb.md)
```

Comment:
FYI , not directly related to this doc but need to be added to the Horizon guide:

When I tried to change the properties using the Horizon console, the first (confusing) step I faced is to find a correct match between the project name and ID displayed on the Manager and the one displayed on Horizon. 
In fact the name that you can see on the Manager is the 'openstack-project-description'. To find it you can use the Identity/project menu on the Horizon Console.